src/components/Nav/Nav.js

```diff
@@ -1,7 +1,108 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import NavModal from './NavModal/NavModal';
+import Navlogin from './NavModal/Navlogin';
+import Navinfo from './NavModal/Navinfo';
+import cart from './icon/cart.svg';
+import info from './icon/info.svg';
 import './Nav.scss';
+import '../../styles/common.scss';
 
 function Nav() {
-  return;
+  const [navlogin, setNavlogin] = useState(false);
+  const [navinfo, setNavinfo] = useState(false);
+  const [currentMenuId, setCurrentMenuId] = useState('');
+  return (
+    <div className="nav">
+      <div className="startHeader">
+        <Link to="/Main">
+          <img className="logo" src="images/nav/Logo.PNG" alt="Logo" />
+        </Link>
+      </div>
+
+      <div className="secondHeader">
+        {NAVBAR_LIST.map(({ title, id }) => {
+          return (
+            <div
+              className="navMenuWrap"
+              key={id}
+              onMouseEnter={() => setCurrentMenuId(id)}
+              onMouseLeave={() => setCurrentMenuId()}
+            >
+              <div className="navMenu">
+                <div className="navMenuTitle"> {title}</div>
+                {currentMenuId === id ? (
+                  <NavModal
+                    setCurrentMenuId={setCurrentMenuId}
+                    currentMenuId={currentMenuId}
+                  />
+                ) : null}
```

Comment:
* 논리연산자 && 을 활용하시면
```suggestion
                {currentMenuId === id && (
                  <NavModal
                    setCurrentMenuId={setCurrentMenuId}
                    currentMenuId={currentMenuId}
                  />
                )}
```
이렇게 할 수 있습니다.
이 [문서](https://ko.javascript.info/logical-operators)를 참고 해 보세요

Reply:
리뷰 반영이 안됐습니다.